config/tests/pulsar_simulate.yaml

```diff
@@ -0,0 +1,264 @@
+###
+# Simple Pulsar simulator: one GPU, dumping beamforming and post-processing buffers to /tmp
```

Comment:
We might want to have a new folder for this, or move this into the `/tests/` folder, or maybe `/tests/helper_config/`.   Evently the files in `/config/tests/` will be run automatically as a part of the GPU unit testing system @james-s-willis is working on.

Reply:
Moved to `/tests/helper_config`